lib/zendesk_apps_support/package.rb

```diff
@@ -225,7 +225,9 @@ def iframe_only?
     private
 
     def legacy_non_iframe_app?
-      @non_iframe ||= locations.values.flat_map(&:values).any? { |l| l == LEGACY_URI_STUB }
+      iframe_urls = locations.values.flat_map(&:values)
+      return true if iframe_urls.empty?
+      @non_iframe ||= iframe_urls.any? { |l| l == LEGACY_URI_STUB }
```

Comment:
we haven't disallowed it, so `all?` is probably more correct than `any?` in this case.